lib/material/material_color.flow

```diff
@@ -165,6 +165,18 @@ export {
 
 	findMColor(color : int) -> MColor;
 	findMColorWithBackground(color : int, background : int) -> MColor;
+
+	colorAddOpacity(color : int) -> int;
+
+	colorDeleteOpacity(color : int) -> int;
+}
+
+colorAddOpacity(color : int) -> int {
+	color + 0x7f000000;
+}
+
+colorDeleteOpacity(color : int) -> int {
```

Comment:
To have this failproof, it is better to use bit operations, so bitAnd with 0xFFFFFF to remove and bitOr with 0x7F000000. Otherwise someone could add or remove twice leading to overflow/underflow and unexpected results.

Reply:
https://github.com/area9innovation/flow9/pull/918